reimburse.py

```diff
@@ -82,6 +82,7 @@ def users_to_reimburse(lookup, users):
     "properties": {
         "email": {"type": "string", "format": "email"},
         "token": {"type": "string"},
+        "day-of": {"type": "boolean"}
```

Comment:
this was required so we can just get the travel reimbursements without redistributing?

Reply:
Yeah. It's a flag for which user status to look for.

On the day of, we only care about those who are checked in, but while waves are going on, only confirmed users matter.

(God why do I still remember that off the top of my head.)